include/dlaf/common/eti.h

```diff
@@ -0,0 +1,34 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2023, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+/// @file
+
+#pragma once
+
+#include <complex>
+
+#include <dlaf/types.h>
+
+#ifdef DLAF_WITH_GPU
+#define DLAF_EXPAND_ETI_SDCZ_DEVICE(ETI_MACRO, KWORD, ...)          \
+  ETI_MACRO(KWORD, double, Device::CPU, __VA_ARGS__);               \
```

Comment:
missing float (it is the source of CI errors for GPU builds...)

Reply:
Yep, it was missing for testing and wasn't meant to be pushed...!